init/main.c

```diff
@@ -262,11 +262,10 @@ early_param("loglevel", loglevel);
 
 #ifdef CONFIG_BLK_DEV_INITRD
 static void * __init get_boot_config_from_initrd(u32 *_size, u32 *_csum)
-{
+{  
```

Comment:
When working with Git, check regularly that you didn't change code that you weren't going to change.
E.g. the whole diff to this file could be reverted, as it is unrelated to your changes.

Reply:
yeah right i am sorry for that.